Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -168,14 +168,14 @@ final class PurchasesOrchestrator {
             manageSubscriptionsHelper: manageSubscriptionsHelper,
             beginRefundRequestHelper: beginRefundRequestHelper,
             storeMessagesHelper: storeMessagesHelper,
+            diagnosticsTracker: diagnosticsTracker,
             winBackOfferEligibilityCalculator: winBackOfferEligibilityCalculator,
             paywallEventsManager: paywallEventsManager,
             webPurchaseRedemptionHelper: webPurchaseRedemptionHelper,
             dateProvider: dateProvider
         )
 
         self._diagnosticsSynchronizer = diagnosticsSynchronizer
-        self._diagnosticsTracker = diagnosticsTracker
```

Comment:
I had to move this to the main `init` because I wanted to use this in some tests that were using that constructor... I think it's fine but lmk if you think otherwise!